src/Realm.C

```diff
@@ -4033,6 +4033,22 @@ Realm::get_noc_usage(
 }
 
 //--------------------------------------------------------------------------
+//-------- get_shifted_grad_op ---------------------------------------------
+//--------------------------------------------------------------------------
+bool
+Realm::get_shifted_grad_op(
```

Comment:
During our `SuppAlg -> Kernel` transition we had discussed removing functionality from Realm and consolidating it in `SolutionOptions`. I think it would be cleaner if we access `get_shifted_grad_op` from `SolutionOptions` directly `realm_.solutionOptions_.get_shifted_grad_op`.